riscv/processor.cc

```diff
@@ -930,6 +941,11 @@ reg_t processor_t::get_csr(int which)
       if (!supports_extension('V'))
         break;
       return VU.vtype;
+    case CSR_VLENB:
+      require_vector_vs;
+      if (!supports_extension('V'))
+        break;
```

Comment:
If you agree with my comment about SSTATUS_VS, then we can remove this if-statement.

Reply:
agree and the 'if' part has been remobed in new PR.